docs/querying/query-context.md

```diff
@@ -66,6 +66,7 @@ See [SQL query context](sql-query-context.md) for other query context parameters
 |`debug`| `false` | Flag indicating whether to enable debugging outputs for the query. When set to false, no additional logs will be produced (logs produced will be entirely dependent on your logging level). When set to true, the following addition logs will be produced:<br />- Log the stack trace of the exception (if any) produced by the query |
 |`setProcessingThreadNames`|`true`| Whether processing thread names will be set to `queryType_dataSource_intervals` while processing a query. This aids in interpreting thread dumps, and is on by default. Query overhead can be reduced slightly by setting this to `false`. This has a tiny effect in most scenarios, but can be meaningful in high-QPS, low-per-segment-processing-time scenarios. |
 |`sqlPlannerBloat`|`1000`|Calcite parameter which controls whether to merge two Project operators when inlining expressions causes complexity to increase. Implemented as a workaround to exception `There are not enough rules to produce a node with desired properties: convention=DRUID, sort=[]` thrown after rejecting the merge of two projects.|
+|`includeTrailerHeader`|`false`|Whether the response can include trailer headers, which are currently used to indicate any error messages in the case of a chunked response, if any.|
```

Comment:
This flag is too broad and may be unnecessary. I say broad because we may start adding trailers for other purposes too (https://github.com/apache/druid/pull/12058). What situations would you want to set it to `false` and not have it on by default? That will help me understand if its required.

Reply:
We just didn't want to change the existing behaviour. That's all.